guides/common/modules/proc_listing-using-satellite-ansible-modules.adoc

```diff
@@ -0,0 +1,45 @@
+[id="listing-using-satellite-ansible-modules_{context}"]
+== Listing the {Project} Ansible Modules
+
+You can list the installed {Project} Ansible modules by navigating to the following directory:
+
+ifeval::["{build}" == "satellite"]
+----
+# ls /usr/share/ansible/collections/ansible_collections/redhat/satellite/plugins/modules/
+----
+
+Alternatively, you can also see the complete list of {Project} Ansible modules and other related information at https://cloud.redhat.com/ansible/automation-hub/redhat/satellite/docs by logging into the Automation Hub.
+
+endif::[]
+
+ifeval::["{build}" != "satellite"]
+----
+# ls /usr/share/ansible/collections/ansible_collections/theforeman/foreman/plugins/modules/
+----
+endif::[]
+
+
+[NOTE]
+====
+At the time of writing, the `ansible-doc -l` command does not list collections yet.
+====
+
+All modules are in the `redhat.satellite` namespace and can be referred to as follows:
+
+
+ifeval::["{build}" == "satellite"]
+----
+$ ansible-doc redhat.satellite.[module_name]
+----
+endif::[]
+
+ifeval::["{build}" != "satellite"]
+----
+$ theforeman.foreman._module_name
+----
+endif::[]
+
+[NOTE]
+====
+All modules require the `server_url`, `username`, and `password` parameters.
+====
```

Comment:
This should not be a note really. Just a regular sentence somewhere where it makes sense. Ansible tells users if it needs more parameters anyway.

Reply:
Ack